maintainers/maintainer-list.nix

```diff
@@ -12779,5 +12779,10 @@
     github = "mbprtpmnr";
     githubId = 88109321;
   };
-
+  jceb = {
```

Comment:
This should be already in master now.

Reply:
I don't see it yet.

Reply:
I thought I merged another PR with it. Apparently not but please sort the entry alphabetically.